### ISSUE TITLE
Include available payload from Rails' Rollbar into browser JS's Rollbar

### ISSUE THREAD

**Issue by esaborit4code** [rollbar/rollbar-gem#608]
I was trying to move Rollbar JS config from an HAML template to [Rails' Rollbar JS config](https://github.com/rollbar/rollbar-gem#integration-with-rollbarjs) and I realized that, contrary to what I thought, neither the environment nor the person data that comes in the server-side errors payload isn't available in browsers JS errors. Unfortunately, I can't define person data payload in the Ruby config file as it only runs once when the app boots up and not in every request.

I guess I'll have to keep the HAML template meanwhile, but I'd really like to move to the Ruby config file as it looks cleaner.

Thanks!

**Comment by jondeandres** [comment]
Hi @esaborit4code,

we added automatic support for the persona data in the rollbar.js integration with the gem in this PR https://github.com/rollbar/rollbar-gem/pull/602. It was merged and released few weeks ago, you can check the changelog https://github.com/rollbar/rollbar-gem/blob/master/CHANGELOG.md#2150. 

You should ensure that you are using the version 2.15.0 of the gem. If you are indeed using it we would appreciate a bit more of information of the problem so we can fix the bug.

**Comment by esaborit4code** [comment]
Hi @jondeandres, thanks for such a quick response!
It totally was my fault, I thought I was logged in but I wasn't, so that's why I couldn't see any person data! I'm sorry! 😅 

Anyway, I've also found some other issues while moving to the Rails config file: 

* Payload's `environment` isn't set and I think it makes sense for it to be the same as Rails Rollbar environment. Now I'm setting it manually (not a big problem as it isn't dynamic).
* I also used to include the request controller and action in the payload, now I can't. I think that it could be solved using `Proc`s that get run in controller's context. It would look like this:
  ```ruby
  payload: {
    controller: proc { params[:controller] },
    action: proc { params[:action] }
  }
  ```

If you think that would be cool, I can try to make some pull requests.

Thanks again!
